src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -749,6 +759,12 @@ public static ComponentResultEntity toEntity(LoadFlowResult.ComponentResult comp
                     .retrieve()
                     .bodyToMono(UUID.class);
         });
+
+        return resultUuid.flatMap(result ->
+                  studyRepository.updateSecurityAnalysisResultUuid(studyName, userId, result)
+                .doOnSuccess(ignored -> emitStudyChanged(studyName, StudyService.UPDATE_TYPE_SECURITY_ANALYSIS_STATUS))
```

Comment:
use the same convention as the rest of the file:
  .doOnSuccess(e -> ...)

Reply:
Corrected